code/modules/food_and_drinks/food/foods/candy.dm

```diff
@@ -644,3 +644,16 @@
 	icon_state = "toolerone"
 	filling_color = "#7D5F46"
 	goal_difficulty = FOOD_GOAL_NORMAL
+
+// ***********************************************************
+// OmniForge Solutions - Candy
```

Comment:
Omniforge is a new corporation you are creating with this PR? If yes then you should probably talk with someone from the lore team, maybe theres alreaady one that would fit as the owner of this chocolate bar, like the vendor company name for example.

Reply:
It was the idea , but it seems like is going to be better to just make it an NT candy or regular candy.